samples/springboot2/native-pet-store/template.yml

```diff
@@ -0,0 +1,32 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: Example Pet Store API written with SpringBoot with the aws-serverless-java-container library
+
+Globals:
+  Api:
+    # API Gateway regional endpoints
+    EndpointConfiguration: REGIONAL
+
+Resources:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

StageName contains invalid characters (Pattern: ^[a-zA-Z0-9-_]+$) at Resources/ServerlessHttpApiApiGatewayDefaultStage/Properties/StageName